.gitignore

```diff
@@ -10,3 +10,4 @@ python/config/config.cfg
 __enamlcache__
 cpp/Picam/projects/vs2010/bin/x64/Debug/Advanced.ilk
 *.cache
+labview/builds/*
```

Comment:
Let's keep this out of the gitignores until we merge. It will be useful for determining that everyone is running the same executable and determining whether or not vi changes need to be pulled or discarded.